engine_modules/citizenship/tasks.py

```diff
@@ -1,6 +1,22 @@
 # -*- coding: utf-8 -*-
-from engine.tasks import OrderResolutionTask
-from engine_modules.citizenship.models import CitizenShipOrder
+from engine.tasks import InitTask, OrderResolutionTask
+from engine_modules.citizenship.models import Citizenship, CitizenshipOrder
+
+
+class CreateCitizenshipTask(InitTask):
+	"""
+	Create the Citizenship object for the turn
+	"""
+	RESOLUTION_ORDER = 0
+
+	def run(self, game):
+		new_citizenships = []
+		past_citizenships = Citizenship.objects.filter(player__game=game, turn=game.current_turn - 1)
+		for past_citizenship in past_citizenships:
+			citizenship = Citizenship(player=past_citizenship.player,
```

Comment:
for now, we never add newline for longline (instead using our editors soft wrap feature)